catalog/spatial/registry/pom.xml

```diff
@@ -158,7 +158,7 @@
             <dependency>
                 <groupId>com.thoughtworks.xstream</groupId>
                 <artifactId>xstream</artifactId>
-                <version>1.4.4</version>
+                <version>1.4.9</version>
```

Comment:
It's definitely a way to do it, I can't really speak to the need to support different versions or not, but this way does leave the flexibility open in case that was needed.